pages/blueprint/index.js

```diff
@@ -311,7 +311,7 @@ class BlueprintPage extends React.Component {
       users = blueprint.customizations.user;
     }
     const pathSuffix = cockpit.location.path[cockpit.location.path.length - 1];
-    const activeKey = ["details", "selected-components", "images"].includes(pathSuffix) ? pathSuffix : undefined;
+    const activeKey = ["customizations", "packages", "images"].includes(pathSuffix) ? pathSuffix : undefined;
```

Comment:
This change should be done when introducing this line in the previous commit and not here https://github.com/weldr/cockpit-composer/pull/752/commits/e9359dbb62a82d8571f55195569b9b99edb0d84a